src/components/Tooltip/Tooltip.st.css

```diff
@@ -93,3 +94,13 @@
     opacity: 0;
     transition: transform 80ms linear, applyOpacity 80ms linear;
 }
+
+.root:skin(error)::popoverContent {
+    background-color: #F64D43;
```

Comment:
I think we can use the `errorColor` from `palette.st.css`, right?

Reply:
Yes. You're right.